catalystwan/models/configuration/feature_profile/sdwan/policy_object/color_list.py

```diff
@@ -0,0 +1,49 @@
+from enum import Enum
+from typing import List
+
+from pydantic import BaseModel, PrivateAttr
+
+from catalystwan.api.configuration_groups.parcel import Global, _ParcelBase
+from catalystwan.models.configuration.feature_profile.sdwan.policy_object.object_list_type import PolicyObjectListType
+
+
+class ColorType(str, Enum):
```

Comment:
already defined numerous times, lets reuse:
`catalystwan/models/common.py::TLOCColorEnum`

Reply:
deleted model and reused `catalystwan/models/common.py::TLOCColorEnum`